pkgs/os-specific/linux/rdma-core/default.nix

```diff
@@ -4,7 +4,7 @@
 } :
 
 let
-  version = "31.2";
+  version = "33.0";
```

Comment:
Can we remove this let and convert it to rec?

Reply:
What would be the advantage of converting it to rec?